templates/ingress.yaml

```
@@ -3,7 +3,7 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ .Release.Name }}-ingress{{ template "enterpriseSuffix" . }}
+  name: {{ .Release.Name }}-st2web-ingress{{ template "enterpriseSuffix" . }}
```

To follow the same naming patterns, the following would look better
```suggestion
  name: {{ .Release.Name }}-st2web{{ template "enterpriseSuffix" . }}-ingress
```

---

Oh, I did not realize there was such pattern. Will make my next commit perfect next time. Thanks a lot @armab !